config/initializers/session_store.rb

```diff
@@ -1,6 +1,3 @@
 # Be sure to restart your server when you modify this file.
 
-Rails.application.config.session_store :cookie_store, {
-  key: '_GraceTunes_session',
-  expire_after: 1.month
-}
+Rails.application.config.session_store :cookie_store, key: '_GraceTunes_session', expire_after: 1.month
```

Comment:
I'm assuming this change is because the new Rails has a different API?

Reply:
more of a ruby syntax change than a rails API change

https://github.com/roidrage/redis-session-store/issues/124

Reply:
Oh are you telling me the cookie expiration was never working!? lol did you find this incidentally?

Reply:
I think it was working. It's just that ruby 3 changed how keyword args work https://www.ruby-lang.org/en/news/2019/12/12/separation-of-positional-and-keyword-arguments-in-ruby-3-0/